java/src/main/java/org/wildfly/openssl/SSL.java

```diff
@@ -528,6 +528,7 @@ public void load() {
     static final long VERSION_1_1_0 = 0x10100000L;
     static final long VERSION_1_1_0_F = 0x1010006fL;
     static final long VERSION_1_1_1 = 0x10101000L;
+    static final long VERSION_3_0_0 = 0x30000000L;
 
```

Comment:
@rmartinc In this file, I think we should also update the `LIBCRYPTO_NAMES` and `LIBSSL_NAMES` to also include the `*-3-x64` versions, i.e.,

```
private static final String[] LIBCRYPTO_NAMES = {"crypto.1.1", "libcrypto-1_1-x64", "libcrypto-3-x64", "crypto", "libeay32", "libcrypto-1_1"};
private static final String[] LIBSSL_NAMES = {"ssl.1.1", "libssl-1_1-x64", "libssl-3-x64", "ssl", "ssleay32", "libssl32", "libssl-1_1"};
```
What do you think?

Reply:
Yep, done! I always use the system props to specify the libraries directly (`org.wildfly.openssl.path.ssl` and `org.wildfly.openssl.path.crypto`) when using non default paths. Modified the two lines with your changes.

Reply:
I have also rebased again just in case!

Reply:
Thanks!